Tests/ControllersTests/FormControllerTest.swift

```diff
@@ -7,54 +7,59 @@ class FormControllerTest: XCTestCase {
     let rawRequest = "GET /form HTTP/1.1\r\nHost:\r\nConnection:Keep-Alive\r\nUser-Agent:chrome\r\nAccept-Encoding:gzip,deflate"
     let request = Request(for: rawRequest)
 
-    let response = FormController().process(request)
+    FormController.formData = ""
+    let response = FormController.process(request)
 
     XCTAssert(response.body == nil)
   }
 
-  func testItSavesDataOnPostReturnsOnGet() {
+  func testItGrabsDataOnPostReturnsOnGet() {
     let rawPostRequest = "POST /form HTTP/1.1\r\nHost:\r\nConnection:Keep-Alive\r\nUser-Agent:chrome\r\nAccept-Encoding:gzip,deflate\r\ndata=fatcat"
     let rawGetRequest = "GET /form HTTP/1.1\r\nHost:\r\nConnection:Keep-Alive\r\nUser-Agent:chrome\r\nAccept-Encoding:gzip,deflate"
     let postRequest = Request(for: rawPostRequest)
     let getRequest = Request(for: rawGetRequest)
 
-    let _ = FormController().process(postRequest)
-    let response = FormController().process(getRequest)
+    FormController.formData = ""
+    let _ = FormController.process(postRequest)
+    let response = FormController.process(getRequest)
```

Comment:
Since I need data to be persisted across multiple requests, I moved them to static variables within the controllers that need them. I'm using the `PersistentDataController` protocol to group these Controllers.

This works in implementation, but my tests are now order dependent unless I clear the contents to isolate each unit test. Is this a fair trade-off?